en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/endpoints/fail-over-endpoints-with-choreo-connect.md

```diff
@@ -0,0 +1,117 @@
+# Fail Over Endpoints With Choreo Connect.
```

Comment:
Also let's reduce the usage of Choreo Connect in the content.  Here we can say Fail over endpoints.